core/store/Cargo.toml

```diff
@@ -32,10 +32,12 @@ name = "trie_bench"
 harness = false
 
 [features]
-default = []
+default = ["rocksdb"]
```

Comment:
why do we need to make this a feature

Reply:
This is so that you could depend on store without requiring rocksdb.  But since other crates in nearcore do i made it default.  I fine to revert this and just make the dep non-optional.

Reply:
Not sure how you do that given the way it is written today. If you want something to depend on store without rocksdb, you need to refactor the crate

Reply:
@bowenwang1996 Reverted!